docs/components/modeler/bpmn/message-events/message-events.md

```diff
@@ -8,6 +8,12 @@ Message events are events which reference a message; they are used to wait until
 
 ![process](assets/message-events.png)
 
+Message events are used if a single process instance needs to wait for a message from a secondary process or an external system. This is typically a single sender to a single recipient relationship (1:1).
```

Comment:
"Typically" here refers to a potential case that there are no recipients, correct? So the message may be sent but nobody is there to receive it.